v1/translate.yaml

```diff
@@ -0,0 +1,254 @@
+info:
+  version: '1.0.0'
+  title: Content Translation API
+  description: Content Translation API
+  termsofservice: https://github.com/wikimedia/restbase
+  license:
+    name: Apache licence, v2
+    url: https://www.apache.org/licenses/LICENSE-2.0
+paths:
+  /machine/{from}/{to}{/provider}:
+    post:
+      tags: ['Translate']
```

Comment:
Okay. It seems the other ones use nouns, so maybe `Translation` here? Can't think of anything more generic that would reflect the scope better, unless we go all the way to things like `Language`. That could then include language identification, spell-checking etc. related services in the future.